### DISCUSSION TITLE
Error that no suitable Python interpreter found

### DISCUSSION THREAD

**Discussion by DustyLoon** [shivammathur/homebrew-php#3087] [Q&A / Help]
Hello! I'm trying to install an older version of PHP:

install shivammathur/php/php@7.2

The installation fails saying there isn't a sufficient PHP interpreter installed:

checking for a BSD-compatible install... /usr/bin/install -c
checking whether build environment is sane... yes
checking for a race-free mkdir -p... ./build-aux/install-sh -c -d
checking for gawk... no
checking for mawk... no
checking for nawk... no
checking for awk... awk
checking whether make sets $(MAKE)... yes
checking whether make supports nested variables... yes
checking whether make supports nested variables... (cached) yes
checking for a Python interpreter with version >= 3.7... none
configure: error: no suitable Python interpreter found

I did install PHP with brew, it shows version 3.12 is installed:

$ python --version
Python 3.12.5

or

$ python3 --version
Python 3.12.5

Do I need to do something I need to do to the original /usr/bin/python to force the install to see the newer version?

**Comment by shivammathur** [comment]
@DustyLoon 

What is your macOS version?

**Comment by DustyLoon** [comment]
It's a really old 10.14 machine that unfortunately can't have the OS upgraded.
